module5/js/script.js

```diff
@@ -83,7 +83,7 @@ document.addEventListener("DOMContentLoaded", function (event) {
 showLoading("#main-content");
 $ajaxUtils.sendGetRequest(
   allCategoriesUrl,
-  [...], // ***** <---- TODO: STEP 1: Substitute [...] ******
+  buildAndShowHomeHTML, // ***** <---- TODO: STEP 1: Substitute [...] ******
```

Comment:
this is correct. I guess you are confused by the passing of a function as a parameter. Remember that in Javascript, everything is an object. Variables are objects. Functions are objects. You are just passing objects as parameter. To understand how passing this functions, you will have to dig into how `ajaxUtils.sendGetRequest` is used. take a look at the ajax-utils.js file to understand